main.go

```diff
@@ -0,0 +1,85 @@
+package main
```

Comment:
The code needs two improvements:

1. All exported types should have a documentation comment beginning with the type name (e.g. `Connection stores a handle to a MongoDB database`). This is to make basic linters pass without complaints.

2. More whitespace is needed for readability. Every logical block of code should be separated by a new line. Let me know if you need suggestions about where to add whitespace.